src/components/views/TableCellContextMenu.tsx

```diff
@@ -118,8 +148,16 @@ export const ContextMenu: React.FC<{
         <PlusIcon /> Insert {columns} to the right
       </ActionButton>
       <Separator />
+      <ActionButton
+        disabled={!headerDir}
```

Comment:
Can you depend on the values returned earlier `const { rows, columns } = getSelectedCellsCount(view.state)` to decide if you want to disable this?

Reply:
updated

Reply:
I see you added a new method `canAddTableHeader` which depends on the row count and intersects with the existing `getSelectedCellsCount`. I would suggest removing `canAddTableHeader` and updating `getSelectedCellsCount` to just return the counts. You can handle the string manipulation separately (outside the method).

Reply:
updated